src/tutorial/microbe_editor/NucleusTutorial.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace Tutorial;
+
+using System;
+using Newtonsoft.Json;
+
+/// <summary>
+///   Prompts the player to eventually place a nucleus for further progression
+/// </summary>
+public class NucleusTutorial : TutorialPhase
+{
+    private const int TriggersOnNthEditorSession = 11;
+
+    private readonly string cellEditorTab = EditorTab.CellEditor.ToString();
+
+    private bool hasNucleus;
+
+    public NucleusTutorial()
+    {
+        CanTrigger = false;
+    }
+
+    public override string ClosedByName => "NucleusTutorial";
+
+    [JsonProperty]
+    private int EditorEntryCount { get; set; }
+
+    public override void ApplyGUIState(MicrobeEditorTutorialGUI gui)
+    {
+        gui.NucleusTutorialVisible = ShownCurrently;
+    }
+
+    public override bool CheckEvent(TutorialState overallState, TutorialEventType eventType, EventArgs args,
+        object sender)
+    {
+        switch (eventType)
+        {
+            case TutorialEventType.MicrobeEditorOrganellePlaced:
+            {
+                var eventArgs = (OrganellePlacedEventArgs)args;
+                var isNucleus = eventArgs.Definition.Name == "Nucleus";
```

Comment:
This compares the *translated* name, which is going to break immediately for someone using any other language than English. Instead the right way is to compare the *internal* name. And for extra safety the nucleus shouldn't be hardcoded. Instead there should be a readonly field to fetch the nucleus definition from SimulationParameters.